composer.json

```diff
@@ -14,13 +14,12 @@
     "repositories": [
         {
             "type": "vcs",
-            "url": "https://github.com/akeneo/pim-community-dev.git",
-            "branch": "master"
+            "url": "https://github.com/akeneo/pim-community-dev.git"
```

Comment:
Is it still needed now that we have pim-community-dev on Packagist?

Reply:
Good catch, i'll try it